packages/client/hmi-client/src/services/jupyter.ts

```diff
@@ -299,4 +299,10 @@ export class KernelSessionManager {
 	disposeMessage(msgId: string) {
 		this.map.delete(msgId);
 	}
+
+	shutdown() {
+		if (this.jupyterSession) {
+			this.jupyterSession.shutdown();
+		}
```

Comment:
Wouldn't `this.jupyterSession?.shutdoww();` do the same thing?

Reply:
changed as suggested